modules/core/src/test/java/io/fluo/recipes/map/CollisionFreeMapIT.java

```diff
@@ -192,6 +261,31 @@ public void testBasic() {
       expectedCounts.remove("food");
 
       Assert.assertEquals(expectedCounts, getComputedWordCounts(fc));
+
+      try (LoaderExecutor loader = fc.newLoaderExecutor()) {
+        loader.execute(new DocumentLoader("0002", "cat cat hamster hamster"));
+      }
+
+      miniFluo.waitForObservers();
+
+      expectedCounts.put("cat", 4l);
+      expectedCounts.put("hamster", 2l);
+
+      Assert.assertEquals(expectedCounts, getComputedWordCounts(fc));
+
+      try (LoaderExecutor loader = fc.newLoaderExecutor()) {
+        loader.execute(new DocumentLoader("0002", "dog hamster"));
+      }
+
+      miniFluo.waitForObservers();
+
+      miniFluo.waitForObservers();
```

Comment:
is there a reason for calling `waitObservers()` twice in a row or it just a typo?